modules/button/react/lib/IconButton.tsx

```diff
@@ -116,6 +113,16 @@ IconButton.Size = {
 
 export default IconButton;
 
+const getIconButtonBorderRadius = (variant: IconButtonVariant) => {
+  switch (variant) {
+    case IconButtonVariant.Square:
+    case IconButtonVariant.SquareFilled:
+      return {borderRadius: borderRadius.m};
+    default:
+      return {borderRadius: borderRadius.circle};
+  }
+};
+
```

Comment:
This was some minor clean up when I was thinking of it as a variant, I can remove this if it's not relevant.